src/main/webapp/data-visualization.js

```diff
@@ -299,3 +305,47 @@ function setStyle(isCountyQuery) {
   const mapsDiv = document.getElementById('map');
   mapsDiv.style.display = 'none';
 }
+
+function initDataTable(isCountyQuery) {
```

Comment:
I left some comments on the table creation functions in data-table but I'm assuming that we can keep that over there and just merge carefully later